templates/base.html

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
@Phinart98, this looks to me like you encountered a merge conflict locally, potentially, and need to clean things up. Does this file run as expected for you locally?